about.html

```diff
@@ -0,0 +1,787 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leader Impact</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick.css" integrity="sha512-wR4oNhLBHf7smjy0K4oqzdWumd+r5/+6QO/vDda76MW5iug4PT7v86FoEkySIJft3XA0Ae6axhIvHrqwm793Nw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick-theme.css" integrity="sha512-6lLUdeQ5uheMFbWm3CP271l14RsX1xtx+J5x2yeIDkkiBpeVTNhTqijME7GgRKKi6hCqovwCoBTlRBEC20M8Mg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body class="about">
+    <script src="https://global.oktacdn.com/okta-auth-js/7.0.0/okta-auth-js.min.js"></script>
+    <div class="alert alert-danger dn" role="alert"></div>
+    <div class="modal fade sign-in-modal"></div>
+    <div id="okta-config" class="hidden"></div>
+
+    <div class="content-parsys responsivegrid">
+      <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+        <div class="container responsivegrid header aem-GridColumn aem-GridColumn--default--12 layout-grid">
+            <div id="container-3bd95d56fa" class="cmp-container">
+              <div class="home-search-overlay aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="search-overlay">
+                  <a href="#" class="close"><img src="./assets/close.svg"></a>
```

Comment:
What is the main difference between `close.svg` and `close-menu.svg`?

Reply:
Close.svg is for the header search bar 
and close-menu.svg was for the hamburger